lmdeploy/model.py

```diff
@@ -483,11 +483,13 @@ def messages2prompt(self,
         box_map = dict(user=self.user,
                        assistant=self.assistant,
                        system=self.system,
-                       environment=self.environment)
+                       environment=self.environment,
```

Comment:
Do we still need "environment"? What's the difference between "environment" and "tool"?

Reply:
> Do we still need "environment"? What's the difference between "environment" and "tool"?

"environment"  is from internlm2. "tool" is from openai.

Reply:
Should we update the user guide?

Reply:
Original user guide works fine.

Reply:
Let's follow the openai API

Reply:
`llama3.1` should also be updated.